src/root/async.c

```diff
@@ -109,7 +114,8 @@ unsigned __stdcall startthread(void *p)
     AsyncRead *aw = (AsyncRead *)p;
 
     //printf("aw->filesdim = %p %d\n", aw, aw->filesdim);
-    for (size_t i = 0; i < aw->filesdim; i++)
+    size_t dim = aw->filesdim;
+    for (size_t i = 0; i < dim; i++)
```

Comment:
@WalterBright Because of the above, I think that adding Ketmar's comment would be misleading. Apart from this, the change is a small optimization.

But if you prove me wrong, I would be happy to comply ;)